test/pytest/test_utils.py

```diff
@@ -63,6 +67,8 @@ def start_torchserve(
     if gen_mar:
         mg.gen_mar(model_store)
     cmd.extend(["--model-store", model_store])
+    if plugin_folder:
+        cmd.extend(["--plugins-path", plugin_folder])
```

Comment:
did you extend torchserve cmd's option?

Reply:
I added plugin option to be passed in as a parameter to the torchserve start command in the test util.

Reply:
This is a discrepancy b/w prod and local. Please simulate the prod use case.